btrdb/utils/general.py

```diff
@@ -74,6 +74,48 @@ def from_nanoseconds(cls, nsec):
                 break
         return cls(pos)
 
+    def for_aligned_windows(self, start, end):
+        """
+        Returns the aligned_windows's fist and last timestamps, as well as the number of windows,
+        based on a given pointwidth set.
+
+        Parameters
+        ----------
+        start : int or datetime like object
+            The start time in nanoseconds for the range to be queried. (see
+            :func:`btrdb.utils.timez.to_nanoseconds` for valid input types)
+
+        end : int or datetime like object
+            The end time in nanoseconds for the range to be queried. (see
+            :func:`btrdb.utils.timez.to_nanoseconds` for valid input types)
+
+
+        Returns
+        -------
+        aligned_start: int
+            First timestamp would be returned by `aligned_windows` that is inclusive of specified start
+            timestamp.
+        aligned_end: int
+            Last timestamp would be returned by `aligned_windows` that is inclusive of specified start
+            timestamp.
+        n_windows: int
+            The number of windows would be returned by `aligned_windows`.
+
+        Examples
+        --------
+        Querying `aligned_windows` of pointwidth of 30.
+
+        >>> start, end = "2016-03-01", "2016-03-02"
+        >>> pointwidth(30).for_aligned_windows(start, end)
+        (1456790399996657664, 1456876798632525824, 80466)
```

Comment:
Can you also include here as comment for this example the human readable times here?

```suggestion
        (1456790399996657664, 1456876798632525824, 80466) # human readable times
```

Reply:
Added comment with human readable timestamps